server/src/tests/samples/namedtuple.py

```diff
@@ -0,0 +1,15 @@
+from typing import NamedTuple
```

Comment:
Please rename this file to namedtuple1.py for consistency.

Reply:
Also, add a header comment that indicates the use of the test sample and internal comments that indicate where errors or warnings should be emitted by the type analyzer.